Controller/Return/Index.php

```diff
@@ -92,6 +92,7 @@ public function execute(): void
     {
         // Receive all params as this could be a GET or POST request
         $redirectResponse = $this->getRequest()->getParams();
+        $storeId = $this->storeManager->getStore()->getId();
```

Comment:
This will most probably return the default store id in that context. Using order object would be a safer object to get the storeId of the payment.

Reply:
I think it should be alright as we are also using it to get the [storeReturnURL](https://github.com/Adyen/adyen-magento2/blob/develop/Gateway/Request/ReturnUrlDataBuilder.php#L68) in the same manner.